k8s/demo/common/sealed-secrets/logstash-secret.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
+    DATA_STORE_URL: AgCRHzkW5QyOLKvtvnWF5yHi4Lr59vBaCbe6B7uGWSx7LiqdwjLfjaa8osox0gELm3q5xwliNcPg6BFjWI7FxYxL+fVQDQkPOIAAj5G5xqRvdaXkYC/g6psBDLNtM4MEPPKqyjypl4GBFHPuFatQSZiFh8+xp4IsNypAbgnppOToCFSL/HX1dQjlBJOtk03ZnKD8TzICw9sVSIZIrc/68TR1oj7ODw/AUPknmg5v3oI8PI6dW7E5v7iWLBVBrLP0pfiaw5maEW6HOgxQK+KByC22g/40Mb2duNV1e0Y/lXJBkPlITJnUWNsYOaFDHhvXzDeILp922ETK90Ozoyt2n0r46XLqKrx5HGPB2oOSLzwXgX/y3lPFrlZJaIIeFHhrOQEVzzJ0BGjHub/fikNPlbU8GIgKVc5ZJpw7WyfYOidoBNYXWv4APBtLGTdHqCX+5nxU95+dxZxw9oKgIIc+9OK0sQyhW+89IzTiyXIdgq/PVGYtS+tN4mBNmNyUEfkLVZWM/CGIfEfZeVMplcoqq6CEOq+NX41utUo5kRyD3oSUVxcaKvY9FnMOG/rfrs3BFZE6+brJAfxNPf+VsxB1NhUFJ99+D1styq4sARXWzz9iFVLmmamNHfGz0DGylfZmK1bzYjuQdtP3KYlR0GvFq4xmbzHYC9xh4QMofn4kQNO3qFJQu9dBp4xacVJpFSW1Id2ErfazJaXS5sj9NqN3EYcTETVJFTWUkz48WWPEMigW7HfvKBqBDnkQfcfTkQaq0wxTaL2f0EM3iyqfOW2jTKwFkj/CBJ3MLJJfuy9KocKvWbcfQix7nWJL79FD9RcyHAL69bxnbUElDJxFzfXcdZr9nzAva7eozDj4
+    DATA_STORE_PASS: AgCdMENU1tGuT4uUciyGLQ14SOE8DjS6MGNa7jkhMCcRlq3ymes78brTYXRoL++G2KZU4/IWAVu9VOD4WNpiedbdHNhsEUJjaq956CCbdBp1B65ZAODoSfoUQbx4slu1CUugI/nVmLbzGIVACOXoMtfw1rKd2CSFzAXUVpqMBFpAK4lZx0GhwJ1FpnNmIPtxOdjADKInYmkBnV/sr1bqbVIWF2KEHnLY5AMptuebl5W7sgNirzw7o5/H9QYBOQvdMrI592P8ofcSm17A/JCRQziqpYf6LrZAklWWkipNK6xqHGQIiw/pcGy5+Qt+FajIaLwryQqVZBF5bEbTnOwkFZ7M9fjCGmKcF0T3qp6TBHTMGPq4a3olgYaNDVTFcaq2qweD0pB0EWFRKL33EZ9S35hjEZW9KxTaITSkPFqUfNs+0i1RPjUp7HOSvWGQJUNsWODTA4BDchh+wyJnEm6w21Sl8ovBp6zaNAU5THGFf902yutTVtsPzErFan2Ue3qqPK7Ix9K+b5kckgmbrIGFJysA5nuMEfB/mB378zz2wjtK/i5kqe7juHZLHOKQOGIDncc57DAZ3fmRqsQjmDBQ07aTSO4BmvR622OAGEu4GO8hnljuK6njPAodKsFRFJdOzrIMfdNCHj8eAywonyEZU4DPBFDzsfKuzo0qVtnYkj4hqFWeJapjoDYj526BYvbkZpMWdIvkbRhkNtpodrV83dZB
+    DATA_STORE_USER: AgAtEA8KWTAiXk8gyVjrKMBbg3apU3JA8ZqcJovfvrYfTAebB6aaqy15zW+RIMuaSzEPHbHnBbtoE8DKTWJPv+Ry6n7BqFUQTL3SKBttADi3johX2bVw0IKh3GDodl1jIdDyegSS6RHWbgYWBWFCPZFnVKGeGx2KvOSa4WyeZbU7/YE7zhDJyzGbSIBuYVd1fVfnkyQA82VtB5+1g0FNP1LxpX9y2lrrkkkw2HLkIz9R0kG4MwNci4LuyAstYYyBB39isU/Fdc5b4VeFRm6CHuvKEmF4JP32xYtdiPUdaGXwV4sfnjutP2jE5L/gqfrEZKKs0asVCcGSqCAns0JLDuv87TnG+hV95e/Mqfy3wFJmoexg2nfrOn1foORmxpv/+KXS1v4fkaHVCcjJaMjq2XH1ja24oGJjHFaU87SY/bHewxD7rHx9NxtkCaSksxaNHBqCg3a7AlRDPAxjNOnkFGW+St9OAR+toi6XBwa6UopW8a6n67TzLIK51LFuAUeAJBGMAki9Ouopx6okRk3+i9aWbXaWzzB8zvVZXUMSLdBTAb+E7MyKkXlaw/droRzvl0f9Z8TZbb8m2QtMtl7Npvj1Clv1xIyy4Hhbd8NjEltJyEBl4Fx8IZO6BXF5WyYyBVKZIg8Cwo7zcQd97Rgy5ccZub4hW48FKniMGdGOoPYtI6BRthSgeU4EMGxr25EDX69RJ1OerVgfzVOJ9uL8JsLtxakb8N2o9563inUoqzUwzYXdhnMkkzYU/nmt
```

Comment:
do you need to add/change a dummy env var so that LS pods are restarted?

Reply:
ummm better to do that.. but we faces some issues with that that's why we have done it manually